utility/src/main/java/org/oppia/util/data/UrlImageParser.kt

```diff
@@ -0,0 +1,73 @@
+package org.oppia.util.data
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.text.Html
+import android.util.Log
+import android.widget.TextView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.request.RequestOptions
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import java.net.URL
+
+//TODO (#169) :  Replace this with exploration asset downloader
```

Comment:
`// TODO (#169) :  Replace this with exploration asset downloader`

Reply:
Done